source/Calamari.Terraform.Tests/TerraformPlanVariableFixture.cs

```diff
@@ -37,7 +37,7 @@ public void SetUp()
             commandLineRunner = Substitute.For<ICommandLineRunner>();
             variables = new CalamariVariables();
             variables.Set(TerraformSpecialVariables.Action.Terraform.PlanJsonOutput, "True");
-            runningDeployment = Substitute.For<RunningDeployment>(variables);
+            runningDeployment = Substitute.For<MockableRunningDeployment>(variables);
```

Comment:
this is so that NSubstitute has a constructor that it can use to create the object.